xwalk_android_tests.gypi

```diff
@@ -26,6 +26,17 @@
       'includes': [ '../build/java.gypi' ],
     },
     {
+      'target_name': 'native_echo_extension',
+      'type': 'loadable_module',
+      'include_dirs': [
+        '..',
+      ],
+      'sources': [
+        'test/android/data/nativeechoextension/native_echo_extension.c',
```

Comment:
In the current RuntimeClinetShell/RuntimeClientEmbeddedShell test, there is already a Java external extension named "echo" as the js interface, see "test/android/data/extensions-config.json". 
"/test/echo_extension.c" defined the same "echo" as the extension name. So, here I redefine "nativeEcho" as the name of the extension to distinguish the one written in Java.  Most of the codes in echo_extension.c and native_echo_extension.c are the same, excepted the name of the js interface.